1-development/list_posts_by_rating/app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  validates_presence_of :title, :description
+  validates_uniqueness_of :title
```

Comment:
Why create this constraint? And, if you created it, why didn't you implement it in the database?

Reply:
It makes sense to also have DB.